app/views/mou/index.html.erb

```diff
@@ -14,12 +16,25 @@
         <%= link_to "support form", help_index_path, class: "govuk-link" %> if you need a copy.
       </p>
     <% end %>
+
+    <% if current_organisation.signed_mou.attached? %>
```

Comment:
A variable for this has been created above

Reply:
Looks like the db lookup is happening 3x here for `current_organisation.signed_mou`. Perhaps this could be put into an instance variable in the controller? Or does `current_organisation.signed_mou` not actually load the object?